x-pack/legacy/plugins/siem/public/components/matrix_histogram/index.tsx

```diff
@@ -189,59 +211,42 @@ export const MatrixHistogramComponent: React.FC<MatrixHistogramProps &
             />
           )}
 
+          <HeaderSection
+            id={id}
+            title={titleWithStackByField}
+            titleSize={titleSize}
+            subtitle={
+              !isInitialLoading &&
+              (totalCount != null && totalCount >= 0 ? subtitleWithCounts : null)
+            }
+          >
+            <EuiFlexGroup alignItems="center" gutterSize="none">
+              <EuiFlexItem grow={false}>
+                {stackByOptions?.length > 1 && (
```

Comment:
nit I do not think that you need `?` because of its definition `stackByOptions: MatrixHistogramOption[];`

Reply:
Alos in line 143